velox/exec/Task.h

```diff
@@ -194,7 +194,9 @@ class Task : public std::enable_shared_from_this<Task> {
   /// of buffers. No more calls are expected after the call with noMoreBuffers
   /// == true, but occasionally the caller might resend it, so calls
   /// received after a call with noMoreBuffers == true are ignored.
-  void updateBroadcastOutputBuffers(int numBuffers, bool noMoreBuffers);
+  /// @return bool Return true if the update was successful, else false. Also,
+  /// returns true if noMoreBroadcastBuffers_.
```

Comment:
noMoreBroadcastBuffers_ is a private member variables; the users of Task may not know how it is set. Consider explaining the API in public terms only: "Returns true without making any changes if a call with noMoreBuffers = true was received earlier."

Perhaps, combine the 2 sentences. "@return true if the update was successful or there was an earlier call with noMoreBuffers == true. Returns false if task failed and buffers were removed."